src/python/CMSSpark/data_aggregation.py

```diff
@@ -0,0 +1,385 @@
+
+    return result
+
+
+def run_agg_aaa(date, fout, ctx, sql_context, verbose=False):
+
+    if verbose:
+        print 'Starting AAA part'
+
+    # Convert date
+    date = short_date_string(date)
+
+    # Use enr instead of raw files
+    hdir = 'hdfs:///project/monitoring/archive/xrootd/enr/gled'
+
+    # Create AAA tables in sql_context
+    aaa_df = aaa_tables(sql_context, hdir=hdir, date=date, verbose=verbose)
+
+    if verbose:
+        print 'Found ' + str(aaa_df['aaa_df'].count()) + ' records in AAA stream'
+
+    # - site name                +
+    # - dataset name             +
+    # - number of access (nacc)  +
+    # - distinct users           +
+    # - stream: aaa              +
+
+    cols = ['src_experiment_site AS site_name',
+            'dataset_name',
+            'count(dataset_name) AS nacc',
+            'count(distinct(aaa_df.user_dn)) AS distinct_users',
+            '\"aaa\" as stream']
+
+
+    # Build a query with "cols" columns
+    query = ("SELECT %s FROM aaa_df " \
+             "JOIN f_b_s_df ON f_b_s_df.file_name = aaa_df.file_lfn " \
+             "GROUP BY src_experiment_site, dataset_name") % ','.join(cols)
+
+    result = run_query(query, sql_context, verbose)
+
+    result = result.sort(desc("nacc"))
+
+    if verbose:
+        print 'Finished AAA part (output is ' + str(result.count()) + ' records)'
+
+    return result
+
+
+def run_agg_cmssw(date, fout, ctx, sql_context, verbose=False):
+
+    if verbose:
+        print 'Starting CMSSW part'
+
+    # Convert date
+    date = long_date_string(date)
+
+    # Create CMSSW tables in sql_context
+    cmssw_df = cmssw_tables(ctx, sql_context, date=date, verbose=verbose)
+
+    if verbose:
+        print 'Found ' + str(cmssw_df['cmssw_df'].count()) + ' records in CMSSW stream'
+
+    # - site name                +
+    # - dataset name             +
+    # - number of access (nacc)  +
+    # - distinct users           +
+    # - stream: cmssw            +
+
+    cols = ['cmssw_df.SITE_NAME AS site_name',
+            'dataset_name',
+            'count(dataset_name) AS nacc',
+            'count(distinct(USER_DN)) AS distinct_users',
+            '\"cmssw\" as stream']
+
+    # Build a query with "cols" columns
+    query = ("SELECT %s FROM cmssw_df "\
+             "JOIN f_b_s_df ON f_b_s_df.file_name = cmssw_df.FILE_LFN "\
+             "GROUP BY cmssw_df.SITE_NAME, dataset_name") % ','.join(cols)
+
+    result = run_query(query, sql_context, verbose)
+
+    result = result.sort(desc("nacc"))
+
+    if verbose:
+        print 'Finished CMSSW part (output is ' + str(result.count()) + ' records)'
+
+    return result
+
+
+def quiet_logs( sc ):
+    print '*** WILL QUIET LOGS ***'
+    logger = sc._jvm.org.apache.log4j
+    logger.LogManager.getRootLogger().setLevel(logger.Level.ERROR)
+    print '*** DID QUIET LOGS ***'
+
+
+def create_file_block_site_table(ctx, sql_context, verbose=False):
```

Comment:
please provide docstring describing function functionality
and use new print function syntax in all print statement.